app/onboarding/page.tsx

```diff
@@ -0,0 +1,170 @@
+'use client';
+
+import React, { useState } from 'react';
+import { ProfileProvider, useProfile } from '@/utils/ProfileProvider';
+
+// Define the possible options for each question
+const states = ['Tennessee', 'Missouri'];
+const gardenTypes = ['Individual', 'Community', 'School'];
+const plotOptions = [
+  { label: 'yes', value: true },
+  { label: 'no', value: false },
+];
+//Interfaces and props to avoid typ errors on Lint
+interface StateSelectionProps {
+  selectedState: string;
+  setSelectedState: React.Dispatch<React.SetStateAction<string>>;
+}
+
+interface GardenTypeSelectionProps {
+  selectedGardenType: string;
+  setSelectedGardenType: React.Dispatch<React.SetStateAction<string>>;
+}
+
+interface PlotSelectionProps {
+  selectedPlot: boolean | null;
+  setSelectedPlot: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+// Select State
+const StateSelection: React.FC<StateSelectionProps> = ({
+  selectedState,
+  setSelectedState,
+}) => {
+  return (
+    <div>
+      <h2>Which state do you live in?</h2>
```

Comment:
i wonder if you can move the heading out of this component and just put it in the retuned HTML? since it's present in every step of this flow

Reply:
ooo yeah i think it might be cleaner to make these reusable components, which take in the props: 
```
{
headerText: string, 
value: string,
setValue: (string) => void,
options: string[],
disabledOption: string,
name: string, 
type: string,
}
```
But that change can be included in the next sprint! Also, since we're hardcoding the value and options to be strings, this structure might only work for StateSelection and GardenTypeSelection, but not PlotSelection? (unless we pass in an optional `optionValues?: boolean[]` prop?)
But even then we also have to map the values of `states` to all caps, (i.e. it should be 'TENNESSEE' in the db) and `gardenTypes` to `UserTypeEnum`

where `export type UserTypeEnum = 'INDIV' | 'SCHOOL' | 'ORG';` should be in the schema